api/auth.js

```diff
@@ -0,0 +1,70 @@
+const express = require('express'),
+  router = express.Router(),
+  db = require('../data/helpers/userModel.js'),
+  bcrypt = require('bcrypt'),
+  { generateJwt } = require('../middleware/auth');
+
+router
+  .post('/register', generateJwt, async function (req, res) {
+    const { user_name, email, password, name } = req.body;
+    const saltRounds = 10;
+
+    if (!user_name || !email || !password || !name) {
+      console.log('Missing information');
+      return res.status(400).json({ error: "Missing information" });
+    }
+
+    console.log('Registering');
+
+    req.body.password = await bcrypt.hash(password, saltRounds).then(hash => hash);
```

Comment:
For this `.then(hash => hash);` if you are just returning what you are getting, do you even need it?